### ISSUE TITLE
List page layouts

### ISSUE THREAD

**Issue by dave-mills** [dave-mills/proinpa#26]
I'm not convinced that the 'list of images' look is ideal for most of the list pages.  It works for the stages, but I'm thinking for others perhaps something along the lines of: 

![item list layout_1](https://cloud.githubusercontent.com/assets/5711101/16151538/701bf87e-3495-11e6-8286-9f7665c62c0e.png)

**Comment by chrismclarke** [comment]
Yes that can work well, ionic has '[cards](http://ionicframework.com/docs/v2/components/#cards)' which are often used to this effect.
We can speak with proinpa on Monday as to what quick access info they think will be best to display and encourage people to read more. Makes sense to have implementation on the roadmap for cop

**Comment by chrismclarke** [comment]
Hadn't realised you already had cards, I've just added row/column layout to the cards to match effect. I'm still not entirely happy with the appearance, can't get the margins quite right (would prefer equal and less padding between cards) and might also experiment with fixed sized pictures (most likely stretching)

**Comment by dave-mills** [comment]
Experiments with fixed-sized pictures underway.  Discovered css stylings "object-fit" and "object-position", which auto-crops images.  See Latest commit (5c98536).

**Comment by dave-mills** [comment]
Task for me related to this: go through and assign some info to fill the space to the right of the images for each item-type.

**Comment by chrismclarke** [comment]
Ooh, haven't played with object-fit and object-position, could be handy. Another thing to keep in mind is when assigning sizes to things its often quite handy to use vw and vh instead of px/em/rem

vw = viewport width (i.e. % of total device width) and vh = viewport height (i.e. % of total device height). You can also play around with things like max-width and max-height to help allow more responsiveness.
